src/symbols.ts

```diff
@@ -380,19 +383,24 @@ async function provideDocumentSymbols(doc: TextDocument): Promise<DocumentSymbol
 			}
 		}
 
+		// Todo clear out symbols from files which are not longer opened in editor.
+		if (!docSymbols.has(doc.fileName)) {
+			docSymbols.set(doc.fileName, new Set<AspSymbol>())
```

Comment:
This can be potential memory leak if you open a lot of files (don't know exactly what the impact will be).